exonum-java-binding/core/rust/exonum-java/src/node.rs

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use java_bindings::{Command, Config, EjbCommand, EjbCommandResult};
+
+use java_bindings::exonum::blockchain::{Blockchain, BlockchainBuilder, InstanceCollection};
+use java_bindings::exonum::exonum_merkledb::{Database, RocksDB};
+use java_bindings::exonum::node::{ApiSender, Node, NodeChannel};
+use java_bindings::exonum::runtime::rust::ServiceFactory;
+use std::sync::Arc;
+
+pub fn run_node(command: Command) -> Result<(), failure::Error> {
+    if let EjbCommandResult::EjbRun(config) = command.execute()? {
+        let node = create_node(config)?;
+        node.run()
+    } else {
+        Ok(())
+    }
+}
+
+fn create_node(config: Config) -> Result<Node, failure::Error> {
+    let channel = NodeChannel::new(&config.run_config.node_config.mempool.events_pool_capacity);
```

Comment:
That's not readable. Please extract in a variable (events_pool_capacity?). Also,
    if you move `node_config` definition above, you can reuse it here to get a shorter chain.

Reply:
Yep, I moved this code a lot to improve readability and didn't spot this line in time. Will fix.